jbug2017 + devconf2019: not using the developer's repo and add resources for conference data being self contained

ochaloup opened the pull request:
Adding resources and updating the README.adoc to have the jbug2017 demo self contained.
This is about to comply the jbug2017 and devconf2019 with the https://github.com/jbosstm/conferences/blob/master/rules.md

ochaloup committed:
[#11] not using the developer's repo and adding resources for jbug2017

tomjenkinson reviewed (commented):
To me, it seems like it's going to be hard to review this. Perhaps I could ask that you can add a few notes about the general steps you took in order to address the issue, please?

ochaloup commented:
@tomjenkinson I understood the issue in way that my conference talk uses the links to my personal repo (github.com/ochaloup) and that links resources which are not part of the conference repo here. I updated the readme and added the missing resources for the quickstart would be self-contained.
I understood it in way that the issue #11 was only about this quickstart as the related document - https://github.com/jbosstm/conferences/blob/master/rules.md - is already updated.
If this PR does not fix the #11 fully then I think it addresses it at least partially. I can remove the `fixes` comment if it's so.

tomjenkinson commented:
I think maybe disconnect this change from #11 and maybe refer to compliance with https://github.com/jbosstm/conferences/blob/master/rules.md

That being said, are the new files in the jbug2017 folder new in the sense that they were not presented then? In which case I wonder if maybe https://github.com/jbosstm/conferences/blob/master/jbug2017/README.adoc should be updated to reflect the changes made after it was presented, I am not speaking to the moving from ochaloup -> jbosstm, rather from the diff like in devconf2019-lra or the parts of jbug2017 that are just those kind of changes. But the mass addition of the files I think it might be useful to say:
"* The quickstart has been updated since it was originally presented. The changes bring it closer in line with https://github.com/jbosstm/conferences/blob/master/rules.md at the time these changes were made"
BTW, if it is fully in line with that file, I am prescribing the wording and if you think it is fully compliant then "closer " can be removed.

tomjenkinson commented:
I also think if we make a link in the README.md to the rules.md then that could address #11

ochaloup committed:
adding info on devconf presentations 2019 + 2021 to README.md

ochaloup commented:
@tomjenkinson I added the note for the jbug2017 and devconf2019 readmes. Plus I found that I forgot to update the root README thus doing so. I removed the link for issue #11  and created a new PR https://github.com/jbosstm/conferences/pull/23 that do the linking the `rules.md` and could be thus fixing the #11.

tomjenkinson commented on devconf2019-lra/README.adoc:
```diff
@@ -6,6 +6,9 @@ Base concept of transaction handling of the LRA specification is https://www.cs.
 Saga pattern is designed to provide transactional guarantees for long running transactions
 which is a good fit for the microservice architecture.
 
+NOTE: The quickstart has been updated since it was originally presented.
```
I may not have been clear before but I don't think it's very necessary to add this `NOTE` to this particular conference if the only changes are to the paths but it's not wrong either so just wanted to confirm it was not what I thought was necessary but keeping it is good - thanks!

tomjenkinson commented on devconf2019-lra/README.adoc:
```diff
@@ -70,8 +73,8 @@ and the coordinator is running you can run this demo app.
-* `git clone https://github.com/ochaloup/devconf2019-lra.git`
-* `cd devconf2019-lra`
+* `git clone https://github.com/jbosstm/conferences`
+* `cd conferences/devconf2019-lra`
```
Nice catch :)

tomjenkinson reviewed (approved):
Thanks Ondra

I didn't verify all the files changes or run the quickstart - if you were expecting that to happen or then we can discuss who would be available to do that.
